Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "xvii"
-# Do not forget bump the version before release. There were breaking changes.
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
Not sure if anyone will notice, but this should be `0.5.0` - there were a bunch of breaking changes (`new` changed type, `into_inner` returns `NonZeroU16`, etc).

It probably would be right to yank `0.4.1` version.

Reply:
I'll be sure to keep an eye on how many customer complaints we get. :p